packages/matplotlib/meta.yaml

```diff
@@ -36,8 +36,8 @@ build:
     rm -rf matplotlib/backends/qt_editor
     rm -rf matplotlib/backends/web_backend
     rm -rf sphinxext
-    cp $PKGDIR/src/fontlist.json matplotlib
-    cp $PKGDIR/src/Humor-Sans-1.0.ttf matplotlib/mpl-data/fonts/ttf/Humor-Sans.ttf
+    cp $PKGDIR/extras/fontlist.json matplotlib
+    cp $PKGDIR/extras/Humor-Sans-1.0.ttf matplotlib/mpl-data/fonts/ttf/Humor-Sans.ttf
```

Comment:
Maybe we could have just kept the src/ folder for this then? No strong option. Anyway, now that you made all the changes, probably no point in reverting.

Reply:
Yeah, matplotlib is an exceptional case where the files should be copied after the package is built, but for now I'm going to put it in the same folder because it seems to be semantically similar to extras.